v2/common/src/main/java/com/google/cloud/teleport/v2/utils/DataStreamClient.java

```diff
@@ -458,4 +535,72 @@ public StandardSQLTypeName convertMysqlToBigQueryColumnType(MysqlColumn column)
       return StandardSQLTypeName.STRING;
     }
   }
+
+  public StandardSQLTypeName convertPostgresqlToBigQueryColumnType(PostgresqlColumn column) {
+    String dataType = column.getDataType().toUpperCase();
```

Comment:
(Technically not related to this PR) datastream does not seem to handle array fields (i.e. the avro files this dataflow job reads from only have null-values in array-valued fields)